src/plugins/telemetry/server/config/telemetry_labels.ts

```diff
@@ -18,9 +18,9 @@ export const labelsSchema = schema.object(
     branch: schema.maybe(schema.string()),
     ciBuildJobId: schema.maybe(schema.string()),
     ciBuildId: schema.maybe(schema.string()),
-    ciBuildNumber: schema.maybe(schema.number()),
+    ciBuildNumber: schema.maybe(schema.string()),
```

Comment:
I couldn't find any info in the PR about why `ciBuldNumer` has been changed from a number to a string, but I assume it's intentional. Just wanted to ask in case it wasn't.

Reply:
@watson Thanks for review ☺️

I struggled over there a little bit since we are providing 
```
`--telemetry.labels.prId=${process.env.GITHUB_PR_NUMBER || ''}`
``` 
as template string under serverArgs it always coerces the number to string. So that's why I updated the schema and based on the schema I've seen it only uses strings and I thought it is accidentally typed number.

Reply:
I'm pretty sure the `schema.number()` will attempt to convert the strings to numbers when it parses the config

Reply:
In these [CI runs](https://buildkite.com/elastic/kibana-pull-request/builds/60447#01823611-f3f6-483e-98ff-401749933160) it just yelled at me